### PR TITLE
Just a bit of cleanup.

### PR DESCRIPTION
**Pull request by ghost**
Visual Studio 2022 refused to work with the solution and tried to delete the .gitignore. Re added it and am now using VSCode.

### CODE REVIEW

**Comment by ALEXWARELLC** [comment]
This has caused my hair to pull itself out just setting it up.

File: Exiled.Loader/Features/LoaderMessages.cs [hunk 1]
```diff
@@ -41,6 +41,7 @@ public static class LoaderMessages
    ▄█    ███   ███    ███  ▄███     ███▄  ███  ███▌    ▄   ███    ███ ███   ▄███
  ▄████████▀    ██████████ ████       ███▄ █▀   █████▄▄██   ██████████ ████████▀
                                                                                 ";
+        //NOTE: This is why we can't have nice things.
```
Comment (louis1706) [inline Exiled.Loader/Features/LoaderMessages.cs:44]: Remove this line

File: Exiled.Loader/Loader.cs [hunk 2]
```diff
@@ -424,17 +420,13 @@ private static bool CheckPluginRequiredExiledVersion(IPlugin<IConfig> plugin)
...
+                    Log.Error($"You're running an older version of Exiled ({Version.ToString(3)})! {plugin.Name} requires version {plugin.RequiredExiledVersion.ToString(3)}.");
 
                     return true;
                 }
                 else if ((requiredVersion.Major < actualVersion.Major) && !LoaderPlugin.Config.ShouldLoadOutdatedPlugins)
                 {
-                    Log.Error(
-                        $"You're running an older version of {plugin.Name} ({plugin.Version.ToString(3)})! " +
-                        $"Its Required Major version is {requiredVersion.Major}, but the actual version is: {actualVersion.Major}. This plugin will not be loaded!");
+                    Log.Error($"You're running an older version of {plugin.Name} ({plugin.Version.ToString(3)})! " + $"Its Required Major version is {requiredVersion.Major}, but the actual version is: {actualVersion.Major}. This plugin will not be loaded!");
```
Comment (louis1706) [inline Exiled.Loader/Loader.cs:429]: Remove this change (it's useless)

File: Exiled.Loader/Loader.cs [hunk 3]
```diff
@@ -424,17 +420,13 @@ private static bool CheckPluginRequiredExiledVersion(IPlugin<IConfig> plugin)
                 // Exiled is outdated
                 if (requiredVersion.Major > actualVersion.Major)
                 {
-                    Log.Error(
-                        $"You're running an older version of Exiled ({Version.ToString(3)})! {plugin.Name} won't be loaded! " +
-                        $"Required version to load it: {plugin.RequiredExiledVersion.ToString(3)}");
+                    Log.Error($"You're running an older version of Exiled ({Version.ToString(3)})! {plugin.Name} requires version {plugin.RequiredExiledVersion.ToString(3)}.");
```
Comment (louis1706) [inline Exiled.Loader/Loader.cs:423]: Remove this change (it's useless)

File: Exiled.Loader/Loader.cs [hunk 4]
```diff
@@ -376,11 +376,7 @@ public void Run(Assembly[] dependencies = null)
 
             EnablePlugins();
 
-            BuildInfoCommand.ModDescription = string.Join(
-                "\n",
-                AppDomain.CurrentDomain.GetAssemblies()
-                    .Where(a => a.FullName.StartsWith("Exiled.", StringComparison.OrdinalIgnoreCase))
-                    .Select(a => $"{a.GetName().Name} - Version {a.GetName().Version.ToString(3)}"));
+            BuildInfoCommand.ModDescription = string.Join("\n", AppDomain.CurrentDomain.GetAssemblies().Where(a => a.FullName.StartsWith("Exiled.", StringComparison.OrdinalIgnoreCase)).Select(a => $"{a.GetName().Name} - Version {a.GetName().Version.ToString(3)}"));
```
Comment (louis1706) [inline Exiled.Loader/Loader.cs:379]: Remove this change (it's useless)

**Review by louis1706 — CHANGES_REQUESTED** [review]
Fix Build Error

**Comment by Misaka-ZeroTwo** [comment]
There's literally no point to make those changes.

**Comment by NotIntense** [comment]
lmfao wtf is this

**Pull request closed without merging**